packages/tools/src/eventDispatchers/shared/getMouseModifier.ts

```diff
@@ -0,0 +1,18 @@
+import { KeyboardBindings as kb } from '../../enums';
+
+/**
+ * Gets the mouse modifier key from a mouse event.
+ * Supports Shift, Ctrl, Alt, in singly and in combinations of 2
+ * Supports Meta singly.
+ */
+const getMouseModifierKey = (evt) =>
```

Comment:
The logic below is too cumbersome and I don't understand what it is doing. Can you please add more docs? So the evt is mouseEvent, it sees if the shiftKey is pressed with mouse? while ctrl and shiftCtrol is there OR alt or shiftAlt, I'm just lost

Reply:
Redid the logic - it is the same logic, just easier to understand as I broke it out.